R/oc_forward.R

```diff
@@ -31,58 +34,89 @@
-#'   precision of the returned result as defined by its geographical extent,
-#'   (i.e. by the extent of the result's bounding box). See the
-#'   \href{https://opencagedata.com/api#confidence}{API documentation} for
+#' @param limit Numeric vector of integer values to determine the maximum
+#'   number of results returned for each \code{placename}. Integer values
+#'   between 1 and 100 are allowed. Default is 10.
+#' @param min_confidence Numeric vector of integer values between 0 and 10
+#'   indicating the precision of the returned result as defined by its
+#'   geographical extent, (i.e. by the extent of the result's bounding box). See
+#'   the \href{https://opencagedata.com/api#confidence}{API documentation} for
 #'   details. Only results with at least the requested confidence will be
-#'   returned.
-#' @param no_annotations A logical vector indicating whether additional
+#'   returned. Default is \code{NULL}).
+#' @param no_annotations Logical vector indicating whether additional
 #'   information about the result location should be returned. \code{TRUE} by
 #'   default, which means that the output will not contain annotations.
-#' @param no_dedupe A logical vector (default \code{FALSE}), when \code{TRUE}
+#' @param no_dedupe Logical vector (default \code{FALSE}), when \code{TRUE}
 #'   the output will not be deduplicated.
-#' @param no_record A logical vector (default \code{FALSE}), when \code{TRUE} no
-#'   log entry of the query is created and the geocoding request is not cached
+#' @param no_record Logical vector (default \code{FALSE}), when \code{TRUE} no
+#'   log entry of the query is created, and the geocoding request is not cached
 #'   by OpenCage.
-#' @param abbrv A logical vector (default \code{FALSE}), when \code{TRUE}
+#' @param abbrv Logical vector (default \code{FALSE}), when \code{TRUE}
 #'   addresses in the \code{formatted} field of the results are abbreviated
 #'   (e.g. "Main St." instead of "Main Street").
-#' @param add_request A logical vector (default \code{FALSE}) indicating whether
+#' @param add_request Logical vector (default \code{FALSE}) indicating whether
 #'   the request is returned again with the results. If the \code{return} value
 #'   is a \code{df_list}, the query text is added as a column to the results.
 #'   \code{json_list} results will contain all request parameters, including the
-#'   API key used! For \code{geojson_list} this is currently ignored by
-#'   OpenCage.
+#'   API key used! This is currently ignored by OpenCage if return value is
+#'   \code{geojson_list}.
 #' @param ... Ignored.
 #'
-#' @return \code{oc_forward} returns, depending on the \code{return} parameter,
-#'   a list with either
+#' @return Depending on the \code{return} argument, \code{oc_forward} returns a
+#'   list with either
 #'   \itemize{
 #'   \item the results as tibbles (\code{"df_list"}, the default),
 #'   \item the results as JSON lists (\code{"json_list"}),
```

Comment:
Reading this again, I am not sure whether it becomes clear that this returns JSON as an R list, as opposed to JSON as a text string (in other words, that the string has already been parsed by `jsonlite::fromJSON()`. Borrowing from `geojsonio::geojson_list`, I'd suggest:
```suggestion
#'   \item the results as JSON specified as a list (\code{"json_list"}),
```
Maybe this also makes sense for the `@param return`?

Reply:
I do think that makes it clearer.